app/components/degrees/view.rb

```diff
@@ -6,7 +6,7 @@ class View < GovukComponent::Base
 
     def initialize(data_model:, show_add_another_degree_button: true, show_delete_button: true, has_errors: false, editable: false)
       @data_model = data_model
-      @degrees = @data_model.degrees
+      @degrees = @data_model.is_a?(DegreesForm) ? @data_model.degree_forms : @data_model.degrees
```

Comment:
@jhenderson had to do this here because we renamed the `degrees` method for `DegreesForm`.